cypress/integration/home.spec.ts

```diff
@@ -1,4 +1,48 @@
-it('should redirect to home page', () => {
-  cy.visit('/');
-  cy.location('pathname').should('eq', '/');
-});
+const getTaskTitleInput = () => cy.get('[data-testid="task-title-input"]');
+const getAddTaskButton = () => cy.get('[data-testid="add-task-button"]');
+const getTitleMsgError = () => cy.get('[data-testid="title-msg-error"]');
+const getModalActionTask = () => cy.get('[data-testid="modal-msg-action-task"]');
+const getButtonCheckedTask = () => cy.get('[data-testid="button-checked-task"]');
+const getTextTitleTask = () => cy.get('[data-testid="text-title-task"]');
+const getButtonRemoveTask = () => cy.get('[data-testid="button-remove-task"]');
+
```

Comment:
I suggest using `beforeEach` to avoid `cy.visit('/');` repetition.

```ts
beforeEach(() => {
  cy.visit('/');
});
```

Reply:
Resolved on cf4041d82f46b410d9200b943ed74dae9c181a96